src/gdrdrv/Makefile

```diff
@@ -18,7 +18,7 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-NVIDIA_SRC_DIR ?= $(shell find /usr/src/nvidia-* -name "nv-p2p.h"|head -1|xargs dirname || echo "NVIDIA_DRIVER_MISSING")
+NVIDIA_SRC_DIR ?= $(shell find $$(find /usr/src/ -type d -a -name "nvidia-*" -prune) -name "nv-p2p.h" -print -quit |xargs dirname || echo "NVIDIA_DRIVER_MISSING")
```

Comment:
See my comment in packages/build-rpm-packages.sh.

Reply:
See above.